RELEASING.md

```diff
@@ -0,0 +1,10 @@
+# Release
+
+The following steps should be followed to release the add-ons:
+ 1. Run the workflow [Prepare Release Add-on](https://github.com/zaproxy/zap-core-help/actions/workflows/prepare-release-add-on.yml)
+    indicating, with a comma separated list, the IDs of the add-ons that should be released (e.g. `help, help_fr_FR`). It creates a
```

Comment:
Is `help` the same as like `all` or is help just `en-GB`?

Reply:
The latter.

Reply:
Do we need some sort of `all` handling?

Reply:
At some point we might need (but if you think it's better to have that now I can added it). Not all of the help add-ons are released to marketplace yet (in fact currently just `help` is being released because of lack of sync with Crowdin).

Reply:
Okay no rush